### PR TITLE
Suggestion form styling

### PR DESCRIPTION
**Pull request by skylerwharton**
- Changed appearance of suggestion form: 
screen shot 2016-11-30 at 11 15 37 am

### CODE REVIEW

File: app/templates/main/index.html [hunk 1]
```diff
@@ -16,10 +16,10 @@
         <div class="left-header">
           <div id="google_translate_element" class="translate"></div>
           <div id="links" class="hidden-xs">
-            <a href="http://www.asylumconnect.org/our-catalog/">ABOUT</a>
-            <a href="http://goo.gl/forms/UjxEbg6H3O">SHARE FEEDBACK</a>
-            <a href="{{ url_for('suggestion.suggest') }}">SUBMIT A RESOURCE</a>
-            <a href="http://www.asylumconnect.org/contact-us/">CONTACT US</a>
+            <a href="http://www.asylumconnect.org/our-cause/">ABOUT</a>
```
Comment (yoninachmany) [inline app/templates/main/index.html:19]: actually the deleted links are the most up-to-date from https://github.com/AsylumConnect/asylumconnect.github.io/pull/37/files

File: tests/test_suggestions_model.py [hunk 2]
```diff
@@ -21,11 +20,11 @@ def tearDown(self):
...
-        s_contact_name = "John Doe"
-        s_contact_number = "123-456-7890"
-        s_contact_email = "taco@time.com"
+        '''Test suggestion of an insertion'''
+        s_text = 'Name: Taco Time \n Address: 123 45th Street'
+        s_contact_name = 'John Doe'
+        s_contact_number = '123-456-7890'
+        s_contact_email = 'taco@time.com'
         s_timestamp = datetime.now(pytz.timezone('US/Eastern'))
         suggestion = Suggestion(suggestion_text=s_text,
```
Comment (yoninachmany) [inline tests/test_suggestions_model.py:30]: update `suggestion_text` name

File: tests/test_suggestions_model.py [hunk 3]
```diff
@@ -53,10 +52,11 @@ def test_edit_suggestion(self):
...
-        s_contact_name = "Anonymous Helper"
-        s_contact_email = "anony@mous.com"
-        s_contact_number = "000-001-0101"
+        s_text = 'The phone number of this establishment is incorrect: it ' \
+                 'should be 212-346-5927'
+        s_contact_name = 'Anonymous Helper'
+        s_contact_email = 'anony@mous.com'
+        s_contact_number = '000-001-0101'
         s_timestamp = datetime.now(pytz.timezone('US/Eastern'))
         suggestion = Suggestion(resource_id=r_added.id, suggestion_text=s_text,
```
Comment (yoninachmany) [inline tests/test_suggestions_model.py:61]: update `suggestion_text` name

**Comment by yoninachmany** [comment]
currently 'additional_information' appears before the contact form and in the contact form - how should we remedy?

**Comment by skylerwharton** [comment]
I think it's fine if our form deviates slightly from AC's current form. My reasoning behind making the additional information separate was that it is logically similar to contact information in that it won't be added to the public-facing resource. To remedy the logical distinction between contact information and additional information, I could change "Your Contact Information" to something like "Additional Information" or "Other Information". Thoughts?

**Comment by yoninachmany** [comment]
ok - i think what we have now is fine - there may be two additional information fields, but it's clear that the second one is part of a different section

**Pull request merged**